app/utils/tailing.js

```diff
@@ -1,13 +1,13 @@
-import $ from 'jquery';
 import { throttle, later } from '@ember/runloop';
 
 export default (function () {
-  function Tailing(window1, tailSelector, logSelector) {
-    this.window = window1;
+  function Tailing(tailSelector, logSelector) {
     this.tailSelector = tailSelector;
     this.logSelector = logSelector;
-    this.position = this.window.scrollTop();
-    this.window.scroll(() => {
+    this.window = window;
+    this.document = document;
```

Comment:
Hmm.. Setting these doesn't make much sense to me, they are available globally everywhere.

Reply:
Yeah good point, probably worth just removing that weirdness while we're at it. Updated to use window and document directly.